Formula/clojure-lsp.rb

```diff
@@ -1,30 +1,19 @@
 class ClojureLsp < Formula
   desc "Language Server (LSP) for Clojure"
-  homepage "https://github.com/snoe/clojure-lsp"
+  homepage "https://github.com/clojure-lsp/clojure-lsp"
   # Switch to use git tag/revision as needed by `lein-git-version`
-  url "https://github.com/snoe/clojure-lsp.git",
-      tag:      "release-20201207T142850",
-      revision: "ab32504073688d507b53e47c354733cd6603bc88"
-  version "20201207T142850"
+  url "https://github.com/clojure-lsp/clojure-lsp/releases/download/2021.02.02-14.02.23/clojure-lsp"
```

Comment:
We don't want to download a pre-built binary.

Reply:
Yeah, we realized that later, @Andre0991 will change that

Reply:
Right, we'll change it.

Btw, we are planning to use graalVM. There will be a script called `native-unix-compile.sh` that will work for all recipes (homebrew, nixOS etc). Is executing this script in this recipe ok?

Reply:
Depends on what the script is doing. As long as it's now downloading anything, and it only touches `buildpath` and `prefix`, it's _probably_ okay.

Reply:
The script just generate the jar with `lein uberjar`, run `gu install native-image` to install graalvm `native-image` and use the native-image to build the binary

Reply:
Is there any other homebrew package using graalvm?

Reply:
No, we have some that use graalvm in their normal build process but use something else in homebrew since graalvm is so poorly documented that it's practically impossible to package from source.

Reply:
Not sure what you mean with "practically impossible", GraalVM is already very documented and there other downstreams of clojure-lsp that have support for building with graal, like `nix` here: https://github.com/NixOS/nixpkgs/pull/111864#pullrequestreview-583169397
So I think the issue here is homebrew doesn't support graal compile from source, right?

Reply:
I'd start with figuring out what's going on in the test block first before we have a discussion about the build system. If the test block doesn't work, then the which build system we use is moot.

Reply:
My guess at the moment is that `clojure-lsp` actually directs output to `stderr`, that's why checking `stdout` gives you nothing.

Reply:
One question, we on clojure-lsp have integration tests now, that test the binary, it uses babashka (that is available on homebrew already), it'd be possible to test it with: `bb ./integration-test/run-all.clj ./clojure-lsp`, the last argument is the binary to test.
Could we use that here on the test phase?
The integration tests contain more concise and precise tests than the way homebrew was testing before

Reply:
>  So I think the issue here is homebrew doesn't support graal compile from source, right?

I've looked into making a GraalVM package/formula before and after a day of searching I didn't manage to get anywhere. But if you can make one I'm sure that'll be much appreciated by the other projects using GraalVM.

Reply:
I'd be great to help, but I don't understand about brew formula (I'm a NixOS user 😅 that's why @Andre0991 is helping me), but probably [follow this is a start](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/graalvm/default.nix) since it builds GraalVM from source :)

Reply:
Is there official documentation from graalvm how to build it? Because the nixOS script seems very specific to nixOS instead of being a generic build script.

Reply:
>Could we use that here on the test phase?

A few thoughts on this:
1. We want our tests to simulate a simple interaction between the user and the package, and integration tests are usually a poor fit for this.
2. I'd rather not replace the existing test for the express purpose of papering over the existing test failure. We should understand why the test is failing first.
3. I would be willing to consider replacing the test, subject to item 1 and provided that we understand why the existing test fails.

Reply:
Ok, we can continue to use the old test, is just that it don't test things like persisting in the sqlite (which may break if not compiled correctly with graalvm etc), we can improve that later I think